docs/sources/operators-guide/configure-agent/language-support/go_pull.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Go (pull mode)"
+menuTitle: "Go (pull mode)"
+description: "Instrumenting Golang applications for continuous profiling"
+weight: 30
+---
+
+# Go (pull mode)
+
+## How to add Golang profiling to your application
+
+Modern observability systems generally fall into two categories: those where services push data, and those where the
+observability system pulls data from services. We find that both approaches are suitable: they both have advantages
+and disadvantages, and one of them can be more desirable than another under certain circumstances.
+
+Pyroscope server can operate in both "pull" and "push" modes. The current implementation makes extensive use of Prometheus
+scrape and service discovery mechanisms.
+
+
+## Supported languages and platforms
+
+Any application that exposes data in [`pprof`](https://github.com/google/pprof/blob/master/doc/README.md) format via HTTP can be set as a remote profiling target.
+
+
+## Scrape configuration
+
+Pyroscope uses exactly the same mechanisms as Prometheus does in order to ensure smooth user experience. Therefore, it
+is configured in almost identical way, and [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config)
+can be used as a reference.
+
+If using Pyroscope you will need to add the following content to your `pyroscope/server.yml` Pyroscope config file. See the [Server config documentation](/docs/server-configuration#configuration-file) for more information on where this config is located by default on your system.
+
+If using Phlare ... TODO
```

Comment:
I would be fine removing the "If using Pyroscope" section above in favor of how to use pull mode and send data to phlare -- @simonswine or @cyriltovena can you help here?